run.sh

```diff
@@ -129,7 +133,8 @@ else
                 --virt-cam $CAMID_VIRT \
                 --relative \
                 --adapt_scale \
-                $@
+                $@ \
+        && $(DOCKER_DEPRECATION_WARNING)
```

Comment:
The error @alievk mentioned comes from here, `$DOCKER_DEPRECATION_WARNING` should be the right syntax. 
Imho it'd be cleaner to directly echo in line 109.